src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -67,19 +68,23 @@ final class PublishMercureUpdatesListener
     /**
      * @param array<string, string[]|string> $formats
      */
-    public function __construct(ResourceClassResolverInterface $resourceClassResolver, IriConverterInterface $iriConverter, ResourceMetadataFactoryInterface $resourceMetadataFactory, SerializerInterface $serializer, array $formats, MessageBusInterface $messageBus = null, callable $publisher = null, ?GraphQlSubscriptionManagerInterface $graphQlSubscriptionManager = null, ?GraphQlMercureSubscriptionIriGeneratorInterface $graphQlMercureSubscriptionIriGenerator = null, ExpressionLanguage $expressionLanguage = null)
-    {
-        if (null === $messageBus && null === $publisher) {
-            throw new InvalidArgumentException('A message bus or a publisher must be provided.');
-        }
-
+    public function __construct(
```

Comment:
This one is a BC break, isn't it?

Reply:
yes, not sure how we can avoid BC break here 🤔 

composer.json allows mercure-bundle: "*", but i think previous versions should conflict with 0.5

Reply:
We can add a conflict if necessary but it will not solve the issue of not adding BC break in this listener.

Reply:
if we are adding a BC layer, i don't think there's a need to add a conflict.

Reply:
Yes, not for this change anyway.

Reply:
Oh I didn't see that this listener was experimental. Sorry, you can indeed add a change like this here.

Reply:
IMO we need a BC layer

Reply:
BC layer added 👍